### ISSUE TITLE
[5] Window section to display information about visualization

### ISSUE THREAD

**Issue by MaxVanDeursen** [vandenheuvel/tribler#122]
As a user, I want some section in the window which displays information so that I understand the general layout of the visual display.

**Comment by vandenheuvel** [comment]
We want to help the user understand our visualization, but are we certain we want to explain this using a question mark with hover? Shouldn't we first think about the different ways in which we can achieve the goal rather than decide on a method now?

**Comment by MaxVanDeursen** [comment]
You're right about that, we would indeed rather think of a method. Let me rephrase this ticket so that it is more generalized.
